src/mscorlib/src/System/Text/EncodingForwarder.cs

```diff
@@ -12,6 +12,10 @@ namespace System.Text
 
     internal static class EncodingForwarder
     {
+#if !BIGENDIAN
+        const int Shift16Shift24 = 256 * 256 * 256 + 256 * 256;
```

Comment:
nit: `(1 << 16) | (1 << 24)` seems less magic, less error prone and more obvious when looking at both name and value